src/cpl_nuopc/wav_import_export.F90

```diff
@@ -506,10 +506,10 @@ subroutine export_fields (gcomp, rc)
        iy  = MAPSF(ISEA,2)
        if (MAPSTA(iy,ix) .eq. 1) then
           ! use hstokes to pass LaSL to POP
-          sw_lamult(jsea)  = LAMULT(ISEA)
-          sw_ustokes(jsea) = USSX(ISEA)
-          sw_vstokes(jsea) = USSY(ISEA)
-          !sw_hstokes(jsea) = LASLPJ(ISEA)
+          sw_lamult(jsea)  = LAMULT(jsea)
```

Comment:
@alperaltuntas - great catch. This corresponds to what CC did - and I did not see that. Its different that what the older ww3 cap does. Thanks for finding this problem. Sorry I introduced it.